### PR TITLE
docs: fix linting errors in angular package format guide

### PR DESCRIPTION
**Pull request by sbrshkappa**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

Issue Number: #47179 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/angular-package-format.md [hunk 1]
```diff
@@ -267,15 +267,15 @@ Not all libraries require such granularity.
 Most libraries with a single logical purpose should be published as a single entrypoint.
 `@angular/core` for example uses a single entrypoint for the runtime, because the Angular runtime is generally used as a single entity.
 
-### Resolution of Secondary Entrypoints
+### Resolution of secondary entrypoints
```
Comment (bob-watson) [inline aio/content/guide/angular-package-format.md:270]: In all the typescript documentation I could find that mentions the term, `entry point` seems to be two words.
I'll add that to the spell checker for future linting, but could you make that correction in the files you edited, please?

File: aio/content/guide/angular-package-format.md [hunk 2]
```diff
@@ -288,20 +288,22 @@ License: MIT
...
 </code-example>
 
-## Partial Compilation
+## Partial compilation
 
 Libraries in the Angular Package Format must be published in "partial compilation" mode.
 This is a compilation mode for `ngc` which produces compiled Angular code that is not tied to a specific Angular runtime version, in contrast to the full compilation used for applications, where the Angular compiler and runtime versions must match exactly.
 
-To partially compile Angular code, use the `"compilationMode"` flag in `"angularCompilerOptions"` in your `tsconfig.json`:
+To partially compile Angular code, use the `"compilationMode"` flag in the `"angularCompilerOptions"` property of your `tsconfig.json`:
```
Comment (bob-watson) [inline aio/content/guide/angular-package-format.md:296]:
```suggestion
To partially compile Angular code, use the `compilationMode` flag in the `angularCompilerOptions` property of your `tsconfig.json`:
```
Code format variable and property names don't need double quotes around. The code format is enough notation.

File: aio/content/guide/angular-package-format.md [hunk 3]
```diff
@@ -310,20 +312,20 @@ If your build pipeline does not use the Angular CLI then refer to the [Consuming
...
 
-### Flattening of ES Modules
+### Flattening of ES modules
 
 The Angular Package Format specifies that code be published in "flattened" ES module format.
 This significantly reduces the build time of Angular applications as well as download and parse time of the final application bundle.
 Please check out the excellent post ["The cost of small modules"](https://nolanlawson.com/2016/08/15/the-cost-of-small-modules) by Nolan Lawson.
 
-The Angular compiler has support for generating index ES module files that can then be used to generate flattened modules using tools like Rollup, resulting in a file format we call Flattened ES Module or FESM.
+The Angular compiler has support for generating index ES module files that can then be used to generate flattened modules using tools like Rollup, resulting in a file format called Flattened ES Module or FESM.
```
Comment (bob-watson) [inline aio/content/guide/angular-package-format.md:321]:
```suggestion
The Angular compiler can generate index ES module files. Tools like Rollup can use these files to generate flattened modules in a *Flattened ES Module* (FESM) file format.
```

File: aio/content/guide/angular-package-format.md [hunk 4]
```diff
@@ -310,20 +312,20 @@ If your build pipeline does not use the Angular CLI then refer to the [Consuming
...
 
 FESM is a file format created by flattening all ES Modules accessible from an entrypoint into a single ES Module.
 It's formed by following all imports from a package and copying that code into a single file while preserving all public ES exports and removing all private imports.
 
-The shortened name "FESM" \(pronounced "phesom"\) can have a number after it such as "FESM5" or "FESM2015".
+The shortened name "FESM" \(pronounced "phe-som"\) can have a number after it such as "FESM5" or "FESM2015" .
```
Comment (bob-watson) [inline aio/content/guide/angular-package-format.md:326]:
```suggestion
The abbreviated name, FESM, pronounced *phe-som*, can be followed by a number such as FESM5 or FESM2015.
```

File: aio/content/guide/angular-package-format.md [hunk 5]
```diff
@@ -310,20 +312,20 @@ If your build pipeline does not use the Angular CLI then refer to the [Consuming
...
 The number refers to the language level of the JavaScript inside the module.
-So a FESM5 file would be ESM+ES5 \(import/export statements and ES5 source code\).
+Accordingly a FESM5 file would be ESM+ES5 \(import/export statements and ES5 source code\).
```
Comment (bob-watson) [inline aio/content/guide/angular-package-format.md:328]:
```suggestion
Accordingly a FESM5 file would be ESM+ES5 and include import/export statements and ES5 source code.
```

File: aio/content/guide/angular-package-format.md [hunk 6]
```diff
@@ -349,51 +351,51 @@ Once the index file \(for example, `my-ui-lib.js`\) is generated by ngc, bundler
 
 #### Note about the defaults in package.json
 
-As of webpack v4 the flattening of ES modules optimization should not be necessary for webpack users, and in fact theoretically we should be able to get better code-splitting without flattening of modules in webpack.
-In practice we still see size regressions when using unflattened modules as input for webpack v4.
-This is why `"module"` and `"es2020"` package.json entries still point to fesm files.
-We are investigating this issue and expect that we'll switch the `"module"` and `"es2020"` package.json entry points to unflattened files when the size regression issue is resolved.
+As of webpack v4 the flattening of ES modules optimization should not be necessary for webpack users, and in fact theoretically it should be possible to get better code-splitting without flattening of modules in webpack.
```
Comment (bob-watson) [inline aio/content/guide/angular-package-format.md:354]:
```suggestion
As of webpack v4, the flattening of ES modules optimization should not be necessary for webpack users. It should be possible to get better code-splitting without flattening of modules in webpack.
```

File: aio/content/guide/angular-package-format.md [hunk 7]
```diff
@@ -349,51 +351,51 @@ Once the index file \(for example, `my-ui-lib.js`\) is generated by ngc, bundler
...
+In practice however size regressions are seen when using unflattened modules as input for webpack v4.
```
Comment (bob-watson) [inline aio/content/guide/angular-package-format.md:355]:
```suggestion
In practice, size regressions can still be seen when using unflattened modules as input for webpack v4.
```

File: aio/content/guide/angular-package-format.md [hunk 8]
```diff
@@ -349,51 +351,51 @@ Once the index file \(for example, `my-ui-lib.js`\) is generated by ngc, bundler
...
+This is why `"module"` and `"es2020"` package.json entries still point to FESM files.
```
Comment (bob-watson) [inline aio/content/guide/angular-package-format.md:356]:
```suggestion
This is why `module` and `es2020` package.json entries still point to FESM files.
```

File: aio/content/guide/angular-package-format.md [hunk 9]
```diff
@@ -349,51 +351,51 @@ Once the index file \(for example, `my-ui-lib.js`\) is generated by ngc, bundler
...
+This issue is being investigated and it is expected for a switch to be made to the `"module"` and `"es2020"` package.json entry points to unflattened files when the size regression issue is resolved.
```
Comment (bob-watson) [inline aio/content/guide/angular-package-format.md:357]:
```suggestion
This issue is being investigated. It is expected to switch the `module` and `es2020` package.json entry points to unflattened files after the size regression issue is resolved.
```

File: aio/content/guide/angular-package-format.md [hunk 10]
```diff
@@ -349,51 +351,51 @@ Once the index file \(for example, `my-ui-lib.js`\) is generated by ngc, bundler
...
+The recommendation is for all packages to claim the side-effect free status by setting the `sideEffects` property to `false`, and that developers follow the [Angular Style Guide](https://angular.io/guide/styleguide) which naturally results in code without non-local side-effects.
 
 More info: [webpack docs on side-effects](https://github.com/webpack/webpack/tree/master/examples/side-effects)
 
-### ES2020 Language Level
+### ES2020 language level
 
 ES2020 Language level is now the default language level that is consumed by Angular CLI and other tooling.
-The Angular CLI will downlevel the bundle to a language level that is supported by all targeted browsers at application build time.
+The Angular CLI should downlevel the bundle to a language level that is supported by all targeted browsers at application build time.
```
Comment (bob-watson) [inline aio/content/guide/angular-package-format.md:376]:
```suggestion
The Angular CLI down-levels the bundle to a language level that is supported by all targeted browsers at application build time.
```

**Review by bob-watson — CHANGES_REQUESTED** [review]
Looks good!
Thanks for the edits.
I left a few requests for edits, but overall it looks pretty good.

File: aio/content/guide/angular-package-format.md (hunk 1 above)
Comment (sbrshkappa) [reply aio/content/guide/angular-package-format.md]: Fixed this to read as "entry points"

**Comment by sbrshkappa** [comment]
@bob-watson Thank you for reviewing! Pushed the changes that you suggested. Let me know if there's anything else.

**Review by alan-agius4 — APPROVED** [review]
LGTM

**Review by bob-watson — APPROVED** [review]
LGTM. Thanks!

Reviewed-for: global-docs-approvers

**Review by AndrewKushnir — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 99be607055cd3ac708bd6c6c2e07da0357a15cef.

**Pull request closed without merging**